packages/story-editor/src/components/panels/document/taxonmies/taxonomies.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { __ } from '@web-stories-wp/i18n';
+/**
+ * Internal dependencies
+ */
+import { useHighlights, states, styles } from '../../../../app/highlights';
+import { SimplePanel } from '../../panel';
+
+function TaxonomiesPanel() {
+  const { highlight, resetHighlight } = useHighlights((state) => ({
+    highlight: state[states.EXCERPT],
+    resetHighlight: state.onFocusOut,
+  }));
+
+  return (
+    <SimplePanel
+      name="taxonomies"
+      title={__('Categories and Tags', 'web-stories')}
+      css={highlight?.showEffect && styles.FLASH}
```

Comment:
we don't need this `css` here since it's not getting highlighted. The highlight structure is used with the checklist and quick action menus. For the checklist: If you click on various issues/thumbnails there it'll take you to parts of the editor. For the quick actions menu (the lil menu to the right of the canvas) if you click on the menu items it'll take you to a spot in a panel. 

This is drinking from a fire hose :)

Reply:
Nice, thanks for the context. Totally makes sense.